code/game/turfs/simulated/outdoors/grass.dm

```diff
@@ -63,10 +65,6 @@ var/list/grass_types = list(
 	. = ..()
 
 /turf/simulated/floor/outdoors/grass/Initialize()
-	if(prob(50))
-		icon_state = "[initial(icon_state)]2"
```

Comment:
Was this code not doing anything or..?

Reply:
It's been replaced by the has_base_range system, which automatically picks a turf determined by the name in files.

Same effect, just does it in the flooring define using the pre-existing system instead of individually in the turf code.

Reply:
Refer to flooring.dm, line 31: https://github.com/PolarisSS13/Polaris/pull/7133/files#diff-539c840e82878c1f430addffa6474fb2R31